app/models/case_log.rb

```diff
@@ -439,6 +439,10 @@ def created_by_name
     created_by&.name
   end
 
+  def is_dpo
```

Comment:
Delegate to `created_by` maybe?

Reply:
Oh, I think the issue with that is in case created_by is nil it throws an error with a delegate

Reply:
Even with `allow_nil: true`?